server_stats.cc

```diff
@@ -1,5 +1,18 @@
 #include "server_stats.h"
 
+void ServerStats::init(ServerConfig server_config) {
+
+  for (const auto path : server_config.allPaths()) {
+    const std::string uri_prefix = path.first;
+    const std::string handler_name = path.second;
+
+    if (handler_name == "StatusHandler") {
+      std::vector<string> uri_and_status = { uri_prefix, "200" };
+      handler_call_distribution_[uri_and_status] = 1;
```

Comment:
Nice catch. I forgot that when we access `/status` for the first time, that is 1 request to StatusHandler`